docs/event_details/simple_action.md

```diff
@@ -0,0 +1,24 @@
+# simple_action
+
+## When the event is called
+
+If you’d like to track action start and action finish. You should use "action_start" events.
```

Comment:
**out-of-scope**: You may consider adding links between those pages to make reading those documents even easier.

```md
If you’d like to track action start and action finish. You should use [action_start](action_start.md) events.
```

Reply:
Ok, I added task.